features/dev/verblijfplaats-binnenland-verblijfadres.feature

```diff
@@ -48,28 +48,28 @@ Functionaliteit: Persoon: verblijfplaats binnenland - verblijfadres
     | verblijfadres.inOnderzoek.huisletter                       | <huisletter in onderzoek>          |
     | verblijfadres.inOnderzoek.huisnummer                       | <huisnr in onderzoek>              |
     | verblijfadres.inOnderzoek.huisnummertoevoeging             | <huisnrtoevoeging in onderzoek>    |
-    | verblijfadres.inOnderzoek.straat                           | <straat in onderzoek>              |
+    | verblijfadres.inOnderzoek.officieleStraatnaam              | <officielestraatnaam io>           |
     | verblijfadres.inOnderzoek.postcode                         | <postcode in onderzoek>            |
-    | verblijfadres.inOnderzoek.korteNaam                        | <korteNaam in onderzoek>           |
+    | verblijfadres.inOnderzoek.korteStraatnaam                  | <korteNaam in onderzoek>           |
     | verblijfadres.inOnderzoek.woonplaats                       | <woonplaats in onderzoek>          |
     | verblijfadres.inOnderzoek.datumIngangOnderzoek.type        | Datum                              |
     | verblijfadres.inOnderzoek.datumIngangOnderzoek.datum       | 2002-07-01                         |
     | verblijfadres.inOnderzoek.datumIngangOnderzoek.langFormaat | 1 juli 2002                        |
 
     Voorbeelden:
```

Comment:
Maak de kolommen wat minder breed door 'in onderzoek' te vervangen door 'io' zoals in enkele kolommen al gedaan is.
Dat maakt dat je misschien niet zijwaarts hoeft te scrollen om de tabel in 1 keer in beeld te krijgen.

Reply:
Volgens mij nam je dit mee in deze PR. Ik maak hier dus geen issue voor aan.

Reply:
Dit is door Melvin al gedaan in features\bevragen\persoon\verblijfplaats\adres\verblijfadres\in-onderzoek.feature

Reply:
Betekent dat dan niet dat dis scenario hier verwijderd mag worden?